test/frontend.js

```diff
@@ -153,23 +135,21 @@ describe('Frontend', function () {
       // TODO Make more complex. Unfortunately bugs in Nightmare prevent the clearing and re-entering of fields
       // those Nightmare bugs seem to be gone now, so TODO it is
     })
-    it('Test Logout and Login', function () {
+    it('Test Logout and Login', async function () {
       this.timeout(10000)
-      return n.use(logout()).use(note('logout -> login to: ' + username)).use(login(username))
+      await n.use(logout()).use(note('logout -> login to: ' + username)).use(login(username))
```

Comment:
why was this added? Can this change be undone? There's no need to make this an async function if we return a promise as before. `await` is only needed when we have multiple asynchronous steps and we need to use the results from one step in another.